static-assets/scripts/admin.js

```diff
@@ -435,8 +435,13 @@
       };
 
       var getUsers = function (site) {
+        const params = {
+          limit: 60,
+          ...(site ? { site } : {})
+        };
+
         adminService
-          .getUsers(site)
```

Comment:
This was not being properly sent, since the internal service receives the params object instead of only the site. That was not causing an issue though (still was incorrect), but with the addition of the limit param the change was needed.